tomography_preprocessing/denoising/care4relion/constants.py

```diff
@@ -0,0 +1,13 @@
+even_suffix = '_even' #The suffix given to even tomograms during split tomogram generation. This will be removed from the names of the output tomograms
+
+predict_config_prefix = 'predict_config' #Name (minus the suffix) of the predict_config.json file
+
+train_data_config_prefix = 'train_data_config' #Name (minus the suffix) of the train_data_config.json file
+
+model_name = 'denoising_model' #Name of the model to be trained
+
+train_config_prefix = 'train_config'
```

Comment:
is split tomogram generation still involved here? 

constants usually have all caps names, not sure if we've been sticking to that